cityofaustin/atd-vz-data
# Update comprehensive costs for 2024 ( to account for inflation )

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/14980

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Checkout this branch locally
2. Replicate the VZD to get a fresh copy of prod data and start up your local database.
3. Run `hasura migrate apply --up 1`
4. Ensure the values were updated correctly
5. Run `hasura migrate apply --down 1` make sure this runs and updates correctly
6. You can also run the VZE and see the new note in the VZE landing page talking about the comp costs.


---
#### Ship list
